schema/core.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://docs.oasis-open.org/openeox/tbd/schema/core.json",
+  "title": "EOL and EOS Information",
+  "description": "A draft schema for representing End-of-Life (EOL) and End-of-Support (EOS) information in OpenEoX.",
+  "type": "object",
+  "$defs": {
+    "schema_t": {
+      "title": "OpenEoX Core schema",
+      "description": "Specifies the schema the JSON object must be valid against.",
+      "type": "string",
+      "enum": [
+        "https://docs.oasis-open.org/openeox/tbd/schema/core.json"
+      ]
+    },
+    "timestamp_t": {
+      "title": "Timestamp",
+      "description": "Contains the timestamp at which the product reaches the specified stage.",
+      "type": "string",
+      "oneOf": [
+        {
+          "format": "date-time"
+        },
+        {
+          "enum": [
+            "tba"
+          ]
+        }
+      ]
+    }
+  },
+  "required": [
+    "$schema",
+    "end_of_life",
+    "end_of_security_support",
+    "last_updated"
+  ],
+  "properties": {
+    "$schema": {
+      "$ref": "#/$defs/schema_t"
+    },
+    "end_of_life": {
+      "title": "End of Life",
+      "description": "Indicates the last day when the particular product is officially supported in any way by the vendor.",
+      "$ref": "#/$defs/timestamp_t"
+    },
+    "end_of_sales": {
+      "title": "End of Sales",
+      "description": "Indicates the last day when a particular product may be ordered by customers from vendor sales channels.",
+      "$ref": "#/$defs/timestamp_t"
+    },
+    "end_of_security_support": {
+      "title": "End of Security Support",
+      "description": "Indicates the last day when the vendor has committed to providing security remediations for the particular product.",
+      "$ref": "#/$defs/timestamp_t"
+    },
+    "last_updated": {
+      "title": "Timestamp of last change",
```

Comment:
I believe it will be great to move the RFC 3339 to the separate definition:

```
"rfc3339Timestamp": {
  "title": "RFC 3339 Timestamp",
  "description": "A timestamp formatted according to RFC 3339.",
  "type": "string",
  "format": "date-time"
}
```

Reply:
Thank you for elaboration!